Project-Service-implementation-Script/main.py

```diff
@@ -390,7 +398,7 @@ def programsFileCheck(filePathAddPgm, accessToken, parentFolder, MainFilePath):
                             "\"Description of the Program\" must not be Empty in \"Program details\" sheet")
                         keywordsPGM = dictDetailsEnv['Keywords'].encode('utf-8').decode('utf-8')
                         entitiesPGM = dictDetailsEnv['Targeted entities at program level'].encode('utf-8').decode('utf-8') if dictDetailsEnv['Targeted entities at program level'] else terminatingMessage("\"Targeted entities at program level\" must not be Empty in \"Program details\" sheet")
-                        districtentitiesPGM = dictDetailsEnv['Targeted district at program level'].encode('utf-8').decode('utf-8')
+                        districtentitiesPGM = dictDetailsEnv['Targeted state at program level'].encode('utf-8').decode('utf-8')
```

Comment:
@MallanagoudaB  does the variable name name make sense if you are storing state name?

Reply:
renamed it to stateEntitiesPGM